tests/test_collections.py

```diff
@@ -465,6 +465,16 @@ def test_to_qgis3d(self, borehole_collection):
         assert outfile.is_file()
         outfile.unlink()
 
+    @pytest.mark.unittest
+    def test_to_kingdom(self, borehole_data):
```

Comment:
Borehole_data is layeredData object, vandaar dat (zie vorige comment) het aanroepen van de collection method niet getest wordt.

Gebruik in plaats daarvan de borehole_collection fixture